airflow/providers/tableau/operators/tableau_refresh_workbook.py

```diff
@@ -15,14 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 import warnings
-from typing import TYPE_CHECKING, Optional
+from typing import Optional
 
 from airflow.models import BaseOperator
 from airflow.providers.tableau.operators.tableau import TableauOperator
-
-if TYPE_CHECKING:
-    from airflow.utils.context import Context
-
+from airflow.utils.context import Context
```

Comment:
Needs to be in `TYPE_CHECKING` or handled a different way so that the Tableau provider doesn't have a minimum Airflow version requirement of 2.2.3.

Reply:
Thanks